src/reducers/accounts.js

```diff
@@ -1,42 +1,34 @@
 // @flow
 
 import { handleActions } from 'redux-actions'
+import { createAccountModel } from '@ledgerhq/wallet-common/lib/models/account'
 
 import every from 'lodash/every'
 import get from 'lodash/get'
 import reduce from 'lodash/reduce'
-import defaultsDeep from 'lodash/defaultsDeep'
-
-import { getDefaultUnitByCoinType, getCurrencyByCoinType } from '@ledgerhq/currencies'
+import type { Account } from '@ledgerhq/wallet-common/lib/types'
 
 import type { State } from 'reducers'
-import type { Account } from 'types/common'
 
 export type AccountsState = Account[]
 const state: AccountsState = []
 
+const accountModel = createAccountModel()
+
 function orderAccountsOperations(account: Account) {
   const { operations } = account
-  operations.sort((a, b) => new Date(b.receivedAt) - new Date(a.receivedAt))
+  operations.sort((a, b) => new Date(b.date) - new Date(a.date))
```

Comment:
same?

btw I think `orderAccountsOperations` can be dropped, we assume account.operations are always correctly sorted in all our current algorithm. maybe you just need to insert at the correct place when you add a new op?

Reply:
good point, but let's move that input to a future commit, as it's not directly related to types changes